resources/dispensers/coke/dispensers.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I think we can eliminate this file. See comment on dispenser and places below.

Reply:
Sure thing!

Reply:
The robot icons could be referenced in the `magni.json` file? Unless you're referring to icons of dispensers?

Reply:
Hi @Yadunund! Yes, sorry for being unclear, I'm referring to dispensers' icons. 
My idea is to have something like this

`dispernsers.json`
```
{
    "coke":{
        "icons":{
            "coke": "/robots/magni/magni.png"
        }, 
    }
}
```
A file just with the icons. Because, as you said, the places and its dispenser are already on each <fleet>.json.

I'll remove this file for now because we don't have any icons for the dispensers. I'm writing this here for just a reminder of how the structure should be when we want to create it again.

Reply:
Ah okay a file for the dispenser icon makes sense. Thanks for letting me know.